i18n/en.toml

```diff
@@ -0,0 +1,47 @@
+[blog]
+  other = "Blog"
+[projects]
+  other = "Projects"
+[404_TITLE]
```

Comment:
This is a matter of taste, but doing it same for all would be good. I would prefer "404_title" (i.e. lowercase)

Reply:
Yeah, its a matter of taste but it's good to have coherence in naming of keys, i'll update them all to lowercase.